pkg/api/dashboard.go

```diff
@@ -337,6 +339,18 @@ func (hs *HTTPServer) deleteDashboard(c *contextmodel.ReqContext) response.Respo
 		return response.Error(500, "Failed to delete dashboard", err)
 	}
 
+	err = hs.bus.Publish(c.Req.Context(), &events.DashboardDeleted{
```

Comment:
This isn't going to work bc the dashboard can be a folder. We would need to dispatch an event that has all of the dashboard ids that were deleted, since we wouldn't be able to look them retroactively.